metadata-ingestion/src/datahub/ingestion/source/redash.py

```diff
@@ -529,6 +536,16 @@ def _emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
                 # Continue producing MCE
                 dashboard_slug = dashboard_response["slug"]
                 dashboard_data = self.client.dashboard(dashboard_slug)
+
+                # Check dashboard queries datasource
+                for widget in dashboard_data["widgets"]:
+                    visualization = widget["visualization"]
```

Comment:
Will visualization field always be populated?

Reply:
Hmmm good point. I've checked and the visualization is not always filled. I'll fix that.

Reply:
Solved at: https://github.com/datahub-project/datahub/pull/4616/commits/2440744b4da3f646fae2e596fe676aa54ce96bee